src/YieldDisburser.sol

```diff
@@ -321,25 +302,6 @@ contract YieldDisburser is OwnableUpgradeable {
      * @return uint256[] Distribution of votes for projects
```

Comment:
this natspec comment block should be removed

Reply:
addressed with a11a89d